vue-app/src/store/index.ts

```diff
@@ -542,6 +543,18 @@ const getters = {
       (getters.isRoundContributionPhase || getters.isRoundReallocationPhase)
     )
   },
+
+  isRecipientRegistryOwner(state: RootState): boolean {
+    return (
+      (state.currentUser &&
+        !!state.recipientRegistryInfo &&
+        isSameAddress(
+          state.currentUser.walletAddress,
+          state.recipientRegistryInfo.owner
+        )) ||
+      false
```

Comment:
Wondering if we could just do:
```js
if(state.currentUser || state.recipientRegistryInfo) {
  return false
}

return isSameAddress(
  state.currentUser.walletAddress,
  state.recipientRegistryInfo.owner
)
```
to improve readability.

Reply:
For sure!